fvm/errors.go

```diff
@@ -285,6 +286,19 @@ func (e *EncodingUnsupportedValueError) Code() uint32 {
 	return errCodeEncodingUnsupportedValue
 }
 
+// UnknownError indicates that an unknown error that occurs during tx execution.
+type UnknownError struct {
```

Comment:
I'm not sure if that's the best name - maybe `InternalError` or some `FVMError` to indicate that it's not transaction fault.

Reply:
updated at d097425